csclient/csclient_test.go

```diff
@@ -1011,9 +1020,9 @@ func (s *suite) TestLog(c *gc.C) {
 
 func (s *suite) TestMacaroonAuthorization(c *gc.C) {
 	ch := storetesting.Charms.CharmArchive(c.MkDir(), "wordpress")
-
-	curl := charm.MustParseReference("utopic/wordpress-42")
-	err := s.store.AddCharmWithArchive(curl, ch)
+	curl := charm.MustParseReference("~charmers/utopic/wordpress-42")
```

Comment:
OK, I get you now. I wanted to leave existing tests testing the thing they already tested. In this case it might have been better to change it as it makes little difference.